xbmc/windowing/gbm/DRMUtils.cpp

```diff
@@ -33,6 +34,9 @@ CDRMUtils::CDRMUtils()
 
 bool CDRMUtils::SetMode(const RESOLUTION_INFO& res)
 {
+  if (!CheckConnector(m_connector->connector->connector_id))
```

Comment:
my only other concern is that this seems like an odd place to call this as no modeset will occur until `SetVideoMode` is called.

Reply:
And not even then, because for DRMAtomic ```SetVideoMode``` will only set a flag and the real modeset is done on next call to FlipPage.

Reply:
However `SetMode` is always called before `SetVideoMode`, imho we could get rid of one method in gbm.
What do you think?

Reply:
possibly. It may add redundant code though.